secure-api-gateway-ob-uk-rs-validation/secure-api-gateway-ob-uk-rs-validation-obie/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/validation/obie/ValidatorsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2020-2024 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.sapi.gateway.ob.uk.rs.validation.obie;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackageClasses = ValidatorsConfiguration.class)
+public class ValidatorsConfiguration {
+}
```

Comment:
Can you remove this please?

We want the module: `secure-api-gateway-ob-uk-rs-validation-obie` to be unaware of any spring wiring. The reason for this is that it allows us to customise how validation is done more easily, this is achieved in the class: `DefaultOBValidationModule`.

If we take the component scan approach then it forces everyone to take the validators that spring finds in this package and prevents customers from plugging in custom validation. Currently, if a customer wishes to customise the validation then they can supply their own validation module.

Reply:
Also remove any Spring annotations that have been added to validators

Reply:
What's your recommended approach on this matter then? Should the beans from DefaultOBValidationModule just be named differently between v3 and v4 or something else? Would that work better?

Reply:
Yes just use different names for the methods, the name of the method will be the bean name which you can use if you need a Qualifier.